maintenance/wikia/sql/dataware-schema.sql

```diff
@@ -14,562 +14,1046 @@
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `video_migration_sanitization` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `city_id` int(11) NOT NULL,
+  `old_title` varchar(255) NOT NULL,
+  `sanitized_title` varchar(255) NOT NULL,
+  `operation_status` char(14) NOT NULL DEFAULT 'UNKNOWN',
+  `operation_time` char(14) NOT NULL,
+  `article_title` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `city_id` (`city_id`,`old_title`),
+  KEY `sanitized_title` (`sanitized_title`)
+) ENGINE=InnoDB AUTO_INCREMENT=28091 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
 
 --
--- Table structure for table `user_login_history_summary`
+-- Table structure for table `video_notmigrated`
 --
 
-SET @saved_cs_client     = @@character_set_client;
-SET character_set_client = utf8;
-CREATE TABLE IF NOT EXISTS `user_login_history_summary` (
-  `user_id` int(8) unsigned NOT NULL,
-  `ulh_timestamp` timestamp NOT NULL default CURRENT_TIMESTAMP,
-  PRIMARY KEY  (`user_id`)
+DROP TABLE IF EXISTS `video_notmigrated`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `video_notmigrated` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `wiki_id` int(11) NOT NULL,
+  `wiki_name` varchar(255) NOT NULL,
+  `video_count` int(11) NOT NULL,
+  `wiki_dbname` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  KEY `vnm_wikiid` (`wiki_id`),
+  KEY `vnm_wikiname` (`wiki_name`)
+) ENGINE=InnoDB AUTO_INCREMENT=16437 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `video_notmigrated2`
+--
+
+DROP TABLE IF EXISTS `video_notmigrated2`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `video_notmigrated2` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `wiki_id` int(11) NOT NULL,
+  `wiki_name` varchar(255) NOT NULL,
+  `wiki_dbname` varchar(255) DEFAULT NULL,
+  `video_count` int(11) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `vnm_wikiid` (`wiki_id`),
+  KEY `vnm_wikiname` (`wiki_name`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `video_postmigrate_undo`
+--
+
+DROP TABLE IF EXISTS `video_postmigrate_undo`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `video_postmigrate_undo` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `wiki_id` int(11) NOT NULL,
+  `entry_table` varchar(255) NOT NULL,
+  `entry_id` varchar(255) NOT NULL,
+  `entry_id_field` varchar(255) NOT NULL,
+  `entry_ns_field` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `vpu_wiki_id` (`wiki_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=798955 DEFAULT CHARSET=latin1;
```

Comment:
@garthwebb: do we need this table? It look like a leftover after videos migration.